.github/workflows/mcp-catalog-sync.yaml

```
@@ -320,11 +462,15 @@ jobs:
       needs.preflight.outputs.mcpserver_files != '[]' &&
       (needs.validate.result == 'success' || needs.validate.result == 'skipped')
     runs-on: gha-runner-scale-set
+    outputs:
+      scan_results: ${{ steps.aggregate.outputs.scan_results }}
```

actionlint (1.7.10)

property "aggregate" is not defined in object type {image: {conclusion: string; outcome: string; outputs: {string => string}}; npm-audit: {conclusion: string; outcome: string; outputs: {string => string}}; pip-audit: {conclusion: string; outcome: string; outputs: {string => string}}; trivy: {conclusion: string; outcome: string; outputs: {string => string}}; trivy-process: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)